primefaces/src/test/java/org/primefaces/util/FileUploadUtilsTest.java

```diff
@@ -161,48 +164,36 @@ public void checkPathTraversal_AbsoluteFile() {
         // Act
         if (os.contains("win")) {
```

Comment:
I don't think this condition is ever filled, maybe used MockedStatic as in getValidFilename_Linux ?

Reply:
This is still needed or the unit test fails on Windows.   Because the path is `/C:/...`